### DISCUSSION TITLE
Winform 使用 sqlite 短时间频繁读写数据库会 触发异常并且导致窗体直接闪退 引发的异常:“System.ObjectDisposedException”(位于 System.Data.SQLite.dll 中)

### DISCUSSION THREAD

**Discussion by lhtuling** [dotnetcore/FreeSql#1057] [General]
![image](https://user-images.githubusercontent.com/44140500/160248682-ef08eec3-c845-4137-a615-c5dd43482cde.png)
多线程频繁读写, 触发异常并且导致窗体直接闪退 并且捕获不到异常
只能在输出窗口里看到异常提示
引发的异常:“System.ObjectDisposedException”(位于 System.Data.SQLite.dll 中)

本人菜鸟请问如何解决,或者最起码不要让窗体闪退!

**Comment by 2881099** [comment]
sqlite 并发写不安全的
